templates/cbt/raw/templates.yaml

```diff
@@ -0,0 +1,24 @@
+dataset: cbt
+subset: raw
+templates:
+  2d9e9c74-550e-4838-8d1d-a804d74828f7: !Template
```

Comment:
Can you remove this template? it's not super interesting from the perspective of supervision (very LM-like)

Reply:
please ignore